### PR TITLE
Add article: Bofbof (intro pwn) Write-Up CTF FCSC 2021  

### PR DESCRIPTION
**Pull request by TristanBALDIT**

### CODE REVIEW

File: src/content/posts/FSCS-2021-Intro-Bofbof.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
+---
+title: "write-up : bofbof (FCSC intro 2021)"
+date: 2024-02-14
+#draft: true
```
Comment (ctmbl) [inline src/content/posts/FSCS-2021-Intro-Bofbof.md:4]: you can remove that

File: src/content/posts/FSCS-2021-Intro-Bofbof.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
+---
+title: "write-up : bofbof (FCSC intro 2021)"
+date: 2024-02-14
+#draft: true
+draft: false
+tags: ["iscsc","blog","posts"]
```
Comment (ctmbl) [inline src/content/posts/FSCS-2021-Intro-Bofbof.md:6]: maybe add better tags? check tags of another WU such as https://iscsc.fr/posts/reversing-alien-gibberish-xlitoni/

File: src/content/posts/FSCS-2021-Intro-Bofbof.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+#draft: true
+draft: false
+tags: ["iscsc","blog","posts"]
+---
+
+## Présentation du challenge
+
+Le challenge contient :  
+`docker-compose.yml` : un fichier de configuration permettant d'accéder à un conteneur docker en local  
+`bofbof` : un executable ELF x64, dynamically linked & not stripped
```
Comment (ctmbl) [inline src/content/posts/FSCS-2021-Intro-Bofbof.md:13]: could you add how to start the docker compose ?

**Review by ctmbl — COMMENTED** [review]
Don't know if these images will work we'll see! some change are required still

File: src/content/posts/FSCS-2021-Intro-Bofbof.md (hunk 3 above)
Comment (ctmbl) [reply src/content/posts/FSCS-2021-Intro-Bofbof.md]: [d85247f](https://github.com/iScsc/blog.iscsc.fr/pull/14/commits/d85247f685357ead81645c6dc23c32b038750d1a)

File: src/content/posts/FSCS-2021-Intro-Bofbof.md (hunk 2 above)
Comment (ctmbl) [reply src/content/posts/FSCS-2021-Intro-Bofbof.md]: [d85247f](https://github.com/iScsc/blog.iscsc.fr/pull/14/commits/d85247f685357ead81645c6dc23c32b038750d1a)

File: src/content/posts/FSCS-2021-Intro-Bofbof.md (hunk 1 above)
Comment (ctmbl) [reply src/content/posts/FSCS-2021-Intro-Bofbof.md]: [d85247f](https://github.com/iScsc/blog.iscsc.fr/pull/14/commits/d85247f685357ead81645c6dc23c32b038750d1a)

File: src/content/posts/bofbof-write-up/index.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+    1217:       e8 14 fe ff ff          call   1030 <puts@plt>
+    121c:       b8 00 00 00 00          mov    $0x0,%eax
+    1221:       c9                      leave
+    1222:       c3                      ret
+    1223:       66 2e 0f 1f 84 00 00    cs nopw 0x0(%rax,%rax,1)
+    122a:       00 00 00
+    122d:       0f 1f 00                nopl   (%rax)
+```
+
+Ici, c'est surtout le `main` qui nous va nous intéresser on remarque premièrement l'appel à `vuln` :
```
Comment (ZynoXelek) [inline src/content/posts/bofbof-write-up/index.md:200]:
```suggestion
Ici, c'est surtout le `main` qui va nous intéresser on remarque premièrement l'appel à `vuln` :
```

**Review by ZynoXelek — APPROVED** [review]
Just a small typo but the article is great!

**Pull request merged**